glustercli/cmd/cli_utils_test.go

```diff
@@ -0,0 +1,16 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestHumanReadable checks if the funtion returns the right size with unit
+// for the input given in MB
+func TestHumanReadable(t *testing.T) {
+	assert.Equal(t, "20 MB", HumanReadable(20))
```

Comment:
There's a disparity in output here. If it's `1.0 GB`, then MB output should be `20.0 MB`. This is something that has to be fixed in the actual code below

Reply:
done